src/popup/index.tsx

```diff
@@ -87,4 +88,24 @@ function IndexPopup() {
   )
 }
 
+  useEffect(() => {
+    const handleKeyPress = async (event) => {
+      if (event.key === 'a') {
+        const { owner, repo } = await GetOwnerRepoName()
+        if (owner && repo) {
+          const isItemInList = repoList.some(
+            (item) => item.owner === owner && item.repo === repo
+          )
+          if (!isItemInList) {
+            setRepoList(prevRepoList => [...prevRepoList, {owner, repo}])
+          }
+        }
+      } else if (event.key === 'c') {
+        setRepoList([])
+      }
+    }
+    window.addEventListener('keydown', handleKeyPress)
+    return () => window.removeEventListener('keydown', handleKeyPress)
+  }, [repoList])
```

Comment:
Move this logic to `src/contents/modal/index.tsx`, where we handle the keydown event 'o'. They should be similar. Thanks!